package.json

```diff
@@ -227,8 +227,8 @@
     "zone.js": "0.8.26"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "0.13.1",
-    "@angular/cli": "7.1.3",
+    "@angular-devkit/build-angular": "0.11.4",
```

Comment:
This dependency is downgrading. Is it ok?

Reply:
Yes, check the released version and dependencies: https://github.com/angular/angular-cli/releases/tag/v7.1.4